src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,38 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_SIZE = 10;
+    private K[] keysArray = (K[]) new Object[ARRAY_SIZE];
+    private V[] valuesArray = (V[]) new Object[ARRAY_SIZE];
+    private int size = 0;
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < size; i++) {
+            if (key == keysArray[i] || key != null && key.equals(keysArray[i])) {
+                valuesArray[i] = value;
+                return;
+            }
+        }
+        keysArray[size] = key;
+        valuesArray[size] = value;
+        size++;
     }
 
     @Override
     public V get(K key) {
+        for (int i = 0; i < size; i++) {
+            if (key == keysArray[i] || key != null && key.equals(keysArray[i])) {
+                if (valuesArray[i] != null) {
+                    return valuesArray[i];
+                }
```

Comment:
```suggestion
                    return valuesArray[i];
```
Null is also a possible value)